src/explore-education-statistics-admin/src/pages/admin-dashboard/AdminDashboardPage.tsx

```diff
@@ -33,24 +29,6 @@ const AdminDashboardPage = () => {
       releaseService.getScheduledReleases(),
     ]);
 
-    const contactResultsByRelease = scheduledReleases.map(release =>
-      preReleaseContactService
```

Comment:
Is this being used at all? Could we get rid of it?

Reply:
`preReleaseContactService.getPreReleaseContactsForRelease(releaseId: string)` is still used by `PrereleaseAccessManagement.tsx`